server/src/main/java/org/tctalent/server/service/db/email/EmailHelper.java

```diff
@@ -171,7 +164,9 @@ public void sendVerificationEmail(User user) throws VerifyEmailSendFailedExcepti
             final Context ctx = new Context();
             ctx.setVariable("partner", partner);
             ctx.setVariable("displayName", displayName);
-            String verificationUrl = (user.getRole() == Role.user ? portalUrl : adminUrl) + "/verify-email?token=" + token;
+            String verificationUrl = "https://tctalent.org/api/" +
+                    (user.getRole() == Role.user ? "portal" : "admin") +
+                    "/user/verify-email/" + token;
```

Comment:
What is this used for in the email? We don't need this. A single api call is all that is needed to verify an email - either from a candidate or an admin. They are both users.